packages/ui-file-drop/src/FileDrop/index.tsx

```diff
@@ -157,34 +171,39 @@ class FileDrop extends Component<FileDropProps, FileDropState> {
   get invalid() {
```

Comment:
```get hasMessages() {
    return (
      (this.props as any).messages && (this.props as any).messages.length > 0
    )
  }
``` 
->
```
 get hasMessages() {
    return (
      this.props.messages && this.props.messages.length > 0
    )
  }
```
why did you leave this one out?

Reply:
I think "(this.props as any)" appears in the code with some intention